### PR TITLE
new translation strings

### PR DESCRIPTION
**Pull request by Dxgamer7405**

### CODE REVIEW

**Comment by Dxgamer7405** [comment]
ignore it, i forgot to clean my commits
image

File: Modules/LoadingScreen.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ public static void Update()
                 }
                 else if (LoadingAnimation == null && visible)
                 {
-                    UpdateLoadingAnimation();
+                    // UpdateLoadingAnimation();
```
Comment (Gurge44) [inline Modules/LoadingScreen.cs:47]: Undo pls

File: Patches/ChatCommandPatch.cs [hunk 2]
```diff
@@ -845,7 +845,7 @@ public static void SendRolesInfo(string role, byte playerId, bool isDev = false,
                 var settings = new StringBuilder();
                 if (Options.CustomRoleSpawnChances.ContainsKey(rl))
                 {
-                    settings.AppendLine($"<size=70%><u>Settings for <{Main.roleColors[rl]}>{roleName}</color>:</u>");
+                    settings.AppendLine($"<size=70%><u>{GetString("SettingsForRoleText")}</u> <{Main.roleColors[rl]}>{roleName}</color>:");
```
Comment (Gurge44) [inline Patches/ChatCommandPatch.cs:848]: Move </u> back lol

Comment (Dxgamer7405) [reply Patches/ChatCommandPatch.cs:848]: image
what? </u> is here already

**Pull request merged**